Formula/r/render.rb

```diff
@@ -0,0 +1,28 @@
+class Render < Formula
+  desc "Command-line interface for Render"
+  homepage "https://github.com/render-oss/cli"
+  url "https://github.com/render-oss/cli/archive/refs/tags/v1.0.0.tar.gz"
+  sha256 "bbc1a37e541c2bbfcb96bbff043fe4f9bcb1e80c70a5224639f1e0a99c0e2cc1"
+  license "Apache-2.0"
+  head "https://github.com/render-oss/cli.git", branch: "main"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
+
+  depends_on "go" => :build
+
+  def install
+    ldflags = %W[
+      -s -w
+      -X github.com/renderinc/cli/pkg/cfg.Version=#{version}
+    ]
+    system "go", "build", *std_go_args(ldflags: ldflags)
+  end
+
+  test do
+    assert_match "Usage:", shell_output("#{bin}/render --help")
+    assert_match version.to_s, shell_output("#{bin}/render --version")
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate tewgoath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
The CLI requires authentication to function, so I have added a test to verify a valid error message is printed when running a command.